main.py

```diff
@@ -0,0 +1 @@
+print("The smart home app running (whoop whoop).")
```

Comment:
```suggestion
print("The smart home app running (whoop whoop).")

```
adds newline at end of file :)